share/spack/gitlab/cloud_pipelines/stacks/windows-vis/spack.yaml

```diff
@@ -6,7 +6,7 @@
 spack:
   view: false
   specs:
-  - vtk~mpi
+  - vtk@9~mpi
```

Comment:
why?

Reply:
To force the latest version, and prevent clingo to build successfully a very old version which could mask issues in concretization.

Reply:
i wouldn't add this since you'll forget to bump it to @10 when that's released. Would instead try to fix the underlying issue that prevents 9 from being picked up directly.

Reply:
No, I mean, the issue is fixed - but I just saw by chance that there was an issue making `@9` unsat. This was forcing erroring out. I can remove if that's preferred.

Reply:
Now I'm fixing the lower bound: `@9:`